scripts/apps/users/views/user-preferences.html

```diff
@@ -117,7 +117,7 @@
                     </div>
                 </div>
             </li>
-            <li class="fieldset terms-box" ng-hide="privileges['planning'] !== 1">
+            <li class="fieldset terms-box" ng-hide="!planning_installed">
```

Comment:
you can do the check via features which should be set on root scope, so `ng-if="features.agenda"` would check if the agenda resource is available from the api